addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -1,6 +1,6 @@
 /** @odoo-module **/
 import { patienceDiff } from './patienceDiff.js';
-import { getRangePosition } from '../utils/utils.js';
+import { closestElement, getRangePosition } from '../utils/utils.js';
```

Comment:
Same remark as in the 16.0 branch regarding the commit message.

Reply:
This comment still applies. Please be more specific in your commit message so it's clear what functional issue your commit addresses.

Reply:
Updated!